plugins/woocommerce/client/legacy/css/twenty-twenty-one.scss

```
@@ -3050,6 +3050,20 @@ a.reset_variations {
 	}
 }
 
+@media only screen and (max-width: 768px) {
+	.woocommerce-message,
+	.woocommerce-error li,
+	.woocommerce-info {
+		padding: 1rem 1.5rem;
+
+		.button {
+			margin-left: 10px;
```

One suggestion I can make about this is to use one of the variables that we have, so we can keep consistency on spacing; for example: `$gap`, `$gap-small`, and `$gap-smaller` could be useful to achieve this (you can see a list of variables in here: `packages/js/internal-style-build/abstracts/_variables.scss`)
```suggestion
			padding: $gap-smaller $gap !important;
```